Karma/Models/Charity.cs

```diff
@@ -34,7 +36,27 @@ public class Charity
         public DateTime DateCreated { get; set; }
         [Required]
         [Display(Name = "Review state")]
-        public Enums.ReviewState ReviewState { get; set; }
+        private Enums.ReviewState _reviewState;
+        [Required]
+        [Display(Name = "Review state")]
+        public Enums.ReviewState ReviewState
+        {
+            get => _reviewState;
+            set
+            {
+                if (value != _reviewState)
+                {
+                    _reviewState = value;
+                    
+                    var args = new CharityStateChangedEventArgs();
+                    args.CharityId = this.Id;
+                    args.ReviewState = this.ReviewState;
+                    args.TimeChanged = DateTime.Now;
```

Comment:
Would be better if `DateTime.UtcNow` was used here, since we store time in UTC format almost everywhere.